### ISSUE TITLE
Profile Page

### ISSUE THREAD

**Issue by DjChart** [th3unkn0n/osi.ig#13]
[+] getting profile ...Traceback (most recent call last):
  File "main.py", line 16, in <module>
    ig = main(user=args["username"])
  File "/data/data/com.termux/files/home/osi.ig/.lib/api.py", line 58, in __init__
    self.get_profile()
  File "/data/data/com.termux/files/home/osi.ig/.lib/api.py", line 66, in get_profile
    self.p_data = self.data['entry_data']['ProfilePage'][0]['graphql']['user']
KeyError: 'ProfilePage'

**Comment by radryry** [comment]
Found a fix. (I know next to nothing about Linux/termux/hacking ect. as a whole so don't judge) I don't know if it will be the same for you but try "2to3 . -w" and that should rewrite all your py2 script to py3. My other tools were fixed aswell but if you want to fix something specific like I did for trape try "2to3 -w trape" sorry if I'm writing all this wrong but I guess I'm a script kiddy?.. w. E that means.
-cowabunga dudes - follow my IG and say hi @rad.ryry
